config.json

```diff
@@ -350,7 +350,12 @@
       "core": true,
       "difficulty": 5,
       "slug": "flatten-array",
-      "topics": null,
+      "topics": [
+      	"arrays",
+	"lists",
+	"loops",
+	"recursion"
```

Comment:
Great suggestions! :) The indentation is a bit wonky though. All the topics should be indented two spaces in from `topics` :) Does that make sense?

Reply:
Yeah! I’ll take that into consideration for future PRs. And yeah I can fix identation, thats not a problem